.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Docker image
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+    paths-ignore:
+      - '.gitignore'
+      - '**.md'
+      - 'test/'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install newer git
+      - run: |
```

Comment:
```suggestion
        run: |
```

Reply:
Typo here I think?

Reply:
whoops. jinx.

Reply:
it's doing smth weird when building the image with docker's action..

```
/usr/bin/docker buildx build --iidfile /tmp/docker-build-push-HO6wTt/iidfile --secret id=GIT_AUTH_TOKEN,src=/tmp/docker-build-push-HO6wTt/tmp-4947-OT445lVOThVI --tag ghcr.io/valhalla/valhalla:latest --metadata-file /tmp/docker-build-push-HO6wTt/metadata-file https://github.com/valhalla/valhalla.git#823f1c924454555873d28fcb09bcac4ddf266d6e
#1 [internal] load git source https://github.com/valhalla/valhalla.git#823f1c924454555873d28fcb09bcac4ddf266d6e
#0 0.025 Initialized empty Git repository in /var/lib/docker/overlay2/vfkb97p0dg7afufrjfuyb3cie/diff/
#0 0.029 fatal: Not a valid object name 823f1c924454555873d28fcb09bcac4ddf266d6e^{commit}
#1 12.00 From https://github.com/valhalla/valhalla
```

Reply:
Wild. Maybe that's https://github.com/marketplace/actions/build-and-push-docker-images#git-context?

Reply:
Oh yeah, looks very relevant, thanks! Only skimmed it, already using manual approach, it's even less lines:) seems to work finally!